<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>220</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Linker</string>
  </property>
  <widget class="QPushButton" name="btn_submit">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>40</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>실행</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbx_pnt">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cbx_link">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lb_pnt">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Select a Point layer</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_link">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Select a Line layer</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbx_pnt_col">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>40</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cbx_link_col">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>100</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lb_col1">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>20</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Tracking Columns (as ID)</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_col2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>80</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Tracking Columns (as ID)</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cb_filter">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string> Filter Point layer (unlinked)</string>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_bufsize">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>140</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string>50</string>
   </property>
   <property name="maxLength">
    <number>6</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>15</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_search">
   <property name="geometry">
    <rect>
     <x>226</x>
     <y>149</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Set Max Access Limit (m) :</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="pgbar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>441</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancel">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>90</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>취소</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
